Automattic/jetpack
# AI Assistant: remove unused code from AI Extension

Janitorial PR.
The removed code was previously used for the AI Extension feature's request-handling logic.
However, it is no longer necessary as the extended block transforms into an AI Assistance block instance, delegating all requesting logic. So, we should remove it.

Part of https://github.com/Automattic/jetpack/issues/31936
Fixes #

## Proposed changes:

* AI Assistant: remove unused code from AI Extension

## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Confirm that the AI Extensions feature works as expected.

* Create an instance of one of the extended blocks
* Request a suggestion
* Confirm it requests the suggestion by transforming to an AI Assistant block instance on the fly
* Confirm the rest of the functionality
